doc/api_ref/pubkey.rst

```diff
@@ -979,8 +1154,13 @@ EMSA from X9.31 (EMSA2 in IEEE 1363).
 Raw EMSA
 """"""""
 
-Sign inputs directly.
-Don't use this unless you know what you are doing.
+Sign inputs directly with no hashing or padding
+
+.. warning::
+
+   This exists as an escape hatch allowing an application to define
+   some protocol-specific padding scheme. Don't use this unless you
+   know what you are doing.
```

Comment:
Thinking out loud: once the `PK_Options` (#4318) are fully operational, we could potentially replace this escape hatch with the ability for a user to create their own subclass of `EMSA` to build application-specific paddings. And then pass it along as `.with_padding(ptr_to_an_object_of_my_emsa)`. (Assuming that `class EMSA` is even public API.)

That makes it more involved to YOLO your own padding, with more boilerplate. But perhaps that's actually a good thing.